#### Question 1-

#### T-test: 
T-test refers to a type of parametric test that is applied to identify, how the means of two sets of data differ from one another when variance is not given.

It is based on Student t-distribution.

Popoluation variance is unkonown

Sample size is small.

Ex-A store wants to improve its sales. The previous sales data shows that the average sale of 30 salesmen was Rs.40 per sale. After some training, the current data showed an average sale of Rs. 60 per transaction.

#### Z-test: 
Z-test implies a hypothesis test which ascertains if the means of two datasets are different from each other when variance is given.

It is based on Normal distribution

Population variance is known.

Sample size is Large.

Ex-Let's say that the mean score of students in a class is greater than 70 with a standard deviation of 10. If a sample of 50 students was selected with a mean score of 80.

#### Question 2- 

1. One-tailed test, as the name suggest is the statistical hypothesis test, in which the alternative hypothesis has a single end. On the other hand, two-tailed test implies the hypothesis test; wherein the alternative hypothesis has dual ends.

2. In the one-tailed test, the alternative hypothesis is represented directionally. Conversely, the two-tailed test is a non-directional hypothesis test.

3. In a one-tailed test, the region of rejection is either on the left or right of the sampling distribution. On the contrary, the region of rejection is on both the sides of the sampling distribution.

4. A one-tailed test is used to ascertain if there is any relationship between variables in a single direction, i.e. left or right. As against this, the two-tailed test is used to identify whether or not there is any relationship between variables in either direction.

5. In a one-tailed test, the test parameter calculated is more or less than the critical value. Unlike, two-tailed test, the result obtained is within or outside critical value.

#### Question 3-

Type 1 Error -
* When Null hypothesis is rejected , when in reality it is True.
* the test result says you have coronavirus, but you actually don't.

Type 2 Error-
* When we retain the Null hypothesis, when in reality it is False.
* the test result says you don't have coronavirus, but you actually do.

#### Question 4-

Bayes' Theorem states that the conditional probability of an event, based on the occurrence of another event, is equal to the likelihood of the second event given the first event multiplied by the probability of the first event.

For example: 
* if we have to calculate the probability of taking a blue ball from the second bag out of three different bags of balls, where each bag contains three different colour balls viz. red, blue, black. In this case, the probability of occurrence of an event is calculated depending on other conditions is known as conditional probability.

* A bag I contains 4 white and 6 black balls while another Bag II contains 4 white and 3 black balls. One ball is drawn at random from one of the bags, and it is found to be black. Find the probability that it was drawn from Bag I.

#### Question 5-

The confidence interval is the range of values that we expect our estimate to fall between a certain percentage of the time if we run our experiment again or re-sample the population in the same way.

* A confidence interval is the mean of our estimate plus and minus the variation in that estimate. This is the range of values we expect our estimate to fall between if we redo our test, within a certain level of confidence.

Calculation of CI:

We survey 100 Brits and 100 Americans about their television-watching habits, and find that both groups watch an average of 35 hours of television per week.The variance in the GB estimate is 100, while the variance in the USA estimate is 25.To calculate the 95% confidence interval, we can simply plug the values into the formula.

For USA-

CI= 35+-1.96*5/root(100)

= 35+-1.96(0.5)

= 35+-0.98

Lower bound= 34.02 

Upper bound = 35.98

For Great Britian:

CI= 35+-1.96*10/root(100)

= 35+-1.96(1)

= 35+-1.96

Lower bound= 33.04 

Upper bound = 36.96

#### Question 6-

Ex-A man is known to speak the truth 2 out of 3 times. He throws a die and reports that the number obtained is a four. Find the probability that the number obtained is actually a four.

Solution-

Let A be the event that the man reports that number four is obtained.

Let E1 be the event that four is obtained and E2 be its complementary event.

Then, P(E1) = Probability that four occurs = 1/6.

P(E2) = Probability that four does not occur = 1- P(E1) = 1 – (1/6) = 5/6.

Also, P(A|E1)= Probability that man reports four and it is actually a four = 2/3

P(A|E2) = Probability that man reports four and it is not a four = 1/3.

By using Bayes’ theorem, probability that number obtained is actually a four, P(E1|A)

By using Bayes’ theorem, probability that number obtained is actually a four, P(E1|A)

P(E1)P(A/E1)/{P(A/E1)+P(E2)P(A/E2)}=2/7



#### Question 7-

In [3]:
import math
mean=50
std=5
ci=0.95
n=50
alpha = 0.05
Upper_bound=mean+1.96*std/math.sqrt(n)
Lower_bound=mean-1.96*std/math.sqrt(n)

In [4]:
Upper_bound

51.385929291125635

In [5]:
Lower_bound

48.614070708874365

Range = 48.614 to 51.385

#### Question 8-

Margin of error, also called confidence interval, tells us how much we can expect our survey results to reflect the views from the overall population.

Margin of error is given by-

Z-score(aplha/2)* sigma/root(n)

where, 

alpha= significance value

sigma= standard deviation

n= population size

#### As, n is is denominator, the larger the value of n i.e population size, the smaller will be the margin of error.

#### Question 9-

In [6]:
import math
x_bar=75
meu=70
sigma=5
alpha=0.05
n=50
z_score=(x_bar-meu)*math.sqrt(n)/sigma

In [7]:
z_score

7.0710678118654755

#### Question 10-

In [8]:
n=50
x_bar_minus_meu=6
s=2.5
t_value=x_bar_minus_meu*math.sqrt(n)/s

In [9]:
t_value

16.970562748477143

From the t-table, when aplha=0.05 and dof=49 , the t-value calculated is 1.677

As, the t_value calculated from the calculation is greater than t-value calculated from t-table, the Null hypothesis is rejected.

#### Question 13-

In [11]:
meu=60
sigma=8
n=50
x_bar=50
ci=0.90
alpha=0.10

z-score corresponding to alpha=0.05 is 1.96.

In [17]:
upper_bound=x_bar+(1.96)*sigma/math.sqrt(n)

In [18]:
upper_bound

52.217486865801014

In [19]:
lower_bound=x_bar-(1.96)*sigma/math.sqrt(n)

In [35]:
lower_bound

47.782513134198986

#### Question 14-

Ho= caffeine has significant effect on reaction time

H1= caffeine does not has significant effect on reaction time

In [22]:
n=30
x_bar=0.25
s=0.05
alpha=0.10

In the question, it is not given the population mean so let us assume the population mean as 0.75.

In [32]:
meu=0.20

from the t-table, with aplha = 0.10 and dof=29, we get t_value as 1.311 

Calculation of t_value:
    

In [33]:
t_value=(x_bar-meu)*math.sqrt(n)/s

In [34]:
t_value

5.477225575051659

As, the t_value is greater than that calculated from the t-table , the Null hypothesis is rejected.